Resources/config/schema/doctrine-1.0.xsd

```diff
@@ -47,6 +47,7 @@
         <xsd:attribute name="port" type="xsd:string" />
         <xsd:attribute name="user" type="xsd:string" />
         <xsd:attribute name="password" type="xsd:string" />
+        <xsd:attribute name="application_name" type="xsd:string" />
```

Comment:
I thought about that, but the GUC variable and the DSN part is called `application_name`. Wouldn't it make more sense for the users?